.github/workflows/build.yml

```
@@ -85,6 +85,9 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: "1.17.5"
+      # ensure that code is formatted
+      - run: GOFMT_OUTPUT="$(gofmt -l `find ./node ./event_database -name '*.go' | grep -v vendor` 2>&1)"; [ -n "$GOFMT_OUTPUT" ] && printf "All the following files are not correctly formatted\n${GOFMT_OUTPUT}\n" && exit 1"
```

You have an errant trailing `"`. Try:

```suggestion
      - run: GOFMT_OUTPUT="$(gofmt -l `find ./node ./event_database -name '*.go' | grep -v vendor` 2>&1)"; [ -n "$GOFMT_OUTPUT" ] && printf "All the following files are not correctly formatted\n${GOFMT_OUTPUT}\n" && exit 1
```